docs/modules/trino/pages/usage_guide/connect_to_trino.adoc

```diff
@@ -0,0 +1,75 @@
+= Connect to Trino
+
+Trino offers a SQL endpoint via a HTTP Api.
+A https://trino.io/docs/current/client/jdbc.html[Open source JDBC driver] does exist as well as a proprietary https://docs.starburst.io/data-consumer/clients/odbc.html[ODBC driver].
+
+Multiple SQL clients can be used to connect to Trino as described below.
+
+== Perquisites
+The Trino server must be reachable from your local computer.
+It is normally secured via HTTPS with a certificated provided by a xref:secret-operator:secretclass.adoc[SecretClass].
+
+As most setups use a self-signed certificate, the easiest way is to disable the Trino cert validation as the following guide will do.
+A production-ready solution would be to extract the `ca.crt` from the SecretClass and provide it to the Trino client.
+
+== Connect with trino-cli
+Please have a look at the https://trino.io/docs/current/client/cli.html[official trino-cli docs].
+Please don't forget to specify the `--insecure` flag, so you command looks something like:
+
+[source,console]
+----
+java -jar ~/Downloads/trino-cli-403-executable.jar --server https://85.215.195.29:8443 --user admin --password --insecure
+----
+
+== Connect with DBeaver
+https://dbeaver.io[DBeaver] is a free multi-platform database tool for all people who need to work with databases.
```

Comment:
```suggestion
https://dbeaver.io[DBeaver] is a free multi-platform database tool for anyone who needs to work with databases.
```

Reply:
Copied that quote from their website but I'm fine with the suggestion ^^